src/blob/header.rs

```diff
@@ -9,10 +9,10 @@ use super::FileName;
 pub(crate) const BLOB_VERSION: u32 = 1;
 pub(crate) const BLOB_MAGIC_BYTE: u64 = 0xdeaf_abcd;
 
-#[derive(Debug, Clone, Serialize, Deserialize)]
+#[derive(Debug, Clone, Serialize, Deserialize, PartialEq)]
 pub(crate) struct Header {
-    magic_byte: u64,
-    version: u32,
+    pub magic_byte: u64,
+    pub version: u32,
```

Comment:
What is the reason to make these fields public? And why only these 2 fields without `flags`?

Reply:
@vovac12  Please, answer this question

Reply:
Made all fields `pub(crate)`